notebooks/official/automl/sdk_automl_text_sentiment_analysis_online.ipynb

```diff
@@ -0,0 +1,951 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    delete_all = True
</pre><p>Don't use globals, see https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb for example cleanup cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>